### PR TITLE
doc: Add ORAS new year blog

### PR DESCRIPTION
**Pull request by FeynmanZhou**
This PR adds a blog to summarize the key progress and highlights in 2022 and look forward to 2023

### CODE REVIEW

File: docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+Initially, the reference types work was incubated under the CNCF [ORAS Artifact manifest](https://github.com/oras-project/artifacts-spec) project. It has been contributed to the OCI Image and Distribution v1.1 specifications in Sep 2022. Now it forms the industry standard and has been adopted by some cloud providers and open-source projects, such as [Azure Container Registry](https://azure.microsoft.com/en-us/products/container-registry) and [Zot registry](https://zotregistry.io/v1.4.3/).
+
+## Diverse evangelism and advocacy
+
+Open-source contributions are not limited to coding. The non-code contributions like blogging, writing documentation, and technical sharing are also important for the ORAS community. It’s so glad to see more and more users and contributors from different organizations sharing their use cases and best practices with ORAS toolings via blog posts or conference presentations this year. You can learn more about their experience from their articles and videos below.
```
Comment (TerryHowe) [inline docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md:62]:
```suggestion
Open-source contributions are not limited to coding. The non-code contributions like blogging, writing documentation, and technical sharing are also important for the ORAS community. It’s so good to see more and more users and contributors from different organizations sharing their use cases and best practices with ORAS toolings via blog posts or conference presentations this year. You can learn more about their experience from their articles and videos below.
```

**Comment by SteveLasker** [comment]
Thanks @FeynmanZhou and @TerryHowe for the review. 
The blog post looks great, capturing the progress on the tech and the community. Kudos to you and your team for really building the community this past year. 
I agree with Terry's changes and looking forward to an LGTM with those minor tweaks.

**Comment by yizha1** [comment]
Thanks @FeynmanZhou , incredibly good content.

**Review by SteveLasker — APPROVED** [review]
LGTM
Move to publish ASAP as the "new year" is quickly fading :)

**Comment by FeynmanZhou** [comment]
@SteveLasker Thanks!
BTW, @shizhMSFT as an ORAS owner now has the management permission of ORAS-www Netlify. We will fix the Netlify CI problem today.

File: docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+- [Docker Hub](https://www.docker.com/blog/announcing-docker-hub-oci-artifacts-support/)
+- [GitHub](https://github.com/)
+- [Google Cloud](https://cloud.google.com/anthos-config-management/docs/how-to/sync-oci-artifacts-from-artifact-registry#oras)
+- [Helm](https://v3.helm.sh/docs/topics/registries/)
+- [Singularity](https://sylabs.io/guides/3.1/user-guide/cli/singularity_push.html)
+- [Microsoft Azure - ACR](https://azure.microsoft.com/en-us/products/container-registry/)
+- [Notary v2](https://github.com/notaryproject/notation)
+- [KubeApps](https://github.com/vmware-tanzu/kubeapps) by VMware Tanzu
+- [VMware Application Catalog](https://tanzu.vmware.com/application-catalog)
+- [UOR Framework](https://universalreference.io/) by Red Hat
```
Comment (afflom) [inline docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md:50]: Thank you for the shout-out! We've renamed UOR Framework to Emporous recently. Might you please change this entry for UOR Framework to "Emporous (Formerly UOR Framework)"?

File: docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+### Blogs
+
+- [Notation signatures as ORAS and OCI artifacts](https://notaryproject.dev/blog/2022/oras-oci-artifacts-notation-signatures/) by [maxgio92](https://github.com/maxgio92) from Clastix
+- [Announcing Docker Hub OCI Artifacts Support](https://www.docker.com/blog/announcing-docker-hub-oci-artifacts-support/) by [MILOS GAJDOS](https://www.docker.com/author/milos-gajdos/) from Docker
+- [ORAS 0.14 and Future: Empower Container Secure Supply Chain](https://oras.land/blog/oras-0.14-and-future/) by Feynman Zhou from Microsoft
+- [ORAS 0.15: A Fully Functional OCI Registry Client](https://oras.land/blog/oras-0.15-a-fully-functional-registry-client/) by Feynman Zhou and Yi Zha from Microsoft
+- [Deploy OCI artifacts and Helm charts the GitOps way with Config Sync](https://cloud.google.com/blog/products/containers-kubernetes/gitops-with-oci-artifacts-and-config-sync) from Google Cloud blogs
+- [Storing ABAP build artifacts in OCI registry](https://blogs.sap.com/2022/07/26/storing-abap-build-artifacts-in-oci-registry/) by [Lars Hvam](https://people.sap.com/lars.hvam) from the SAP community blog
+
+### Presentations at conferences
```
Comment (afflom) [inline docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md:73]: Might you please add the below entry to the "Presentations at conferences" section:

```
[Unleashing the Power of the Container Registry](https://www.youtube.com/watch?v=6fohG10y3z0&t=2002s) at DevConf.us by Andrew Block & Alex Flom
```
cc @sabre1041

**Review by TerryHowe — APPROVED** [review]

File: docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md (hunk 3 above)
Comment (FeynmanZhou) [reply docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md:73]: @afflom Good point. Added to the list above.

File: docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md (hunk 2 above)
Comment (FeynmanZhou) [reply docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md]: Renamed to "Emporous (Formerly UOR Framework)"

**Comment by FeynmanZhou** [comment]
The Netlify problem has been fixed. Now it's available to preview it https://deploy-preview-75--oras-project.netlify.app/blog/oras-looking-back-at-2022-and-forward-to-2023/

File: docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md (hunk 3 above)
Comment (afflom) [reply docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md:73]: Thanks so much! @FeynmanZhou

**Review by afflom — APPROVED** [review]

**Review by toddysm — APPROVED** [review]

File: docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+- [VMware Application Catalog](https://tanzu.vmware.com/application-catalog)
+- [Emporous (Formerly UOR Framework)](https://universalreference.io/) by Red Hat
+- [soci-snapshotter](https://github.com/awslabs/soci-snapshotter) by AWS
+- [Zot](https://github.com/project-zot/zot)
+
+## Contributions to upstream OCI
+
+Just a few years ago, there were no standards nor tooling for registries to natively store, discover, and pull a graph of OCI artifacts. To extend the registry’s role and form the industry standard, ORAS maintainers proposed a new artifact manifest type to describe and query relationships between objects stored in a registry, without mutating the existing content.
+
+Initially, the reference types work was incubated under the CNCF [ORAS Artifact manifest](https://github.com/oras-project/artifacts-spec) project. It has been contributed to the OCI Image and Distribution v1.1 specifications in Sep 2022. Now it forms the industry standard and has been adopted by some cloud providers and open-source projects, such as [Azure Container Registry](https://azure.microsoft.com/en-us/products/container-registry) and [Zot registry](https://zotregistry.io/v1.4.3/).
```
[the remark this answers is not in the document]
Comment (SteveLasker) [reply docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md:58]: Agreed with Toddy. Adding some language to encourage folks to join would shift it from "here's the ones that have", to hear some of the early adopters which we're excited to see more join as all registries can support promoting artifacts across them.

[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md]: Good catch. I have incorporated @toddysm 's suggestion.

**Review by SteveLasker — APPROVED** [review]
Thanks @FeynmanZhou 
LGTM

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**